src/wp-admin/includes/class-language-pack-upgrader.php

```diff
@@ -288,6 +288,45 @@ public function bulk_upgrade( $language_updates = array(), $args = array() ) {
 			)
 		);
 
+		foreach ( $language_updates_results as $translation ) {
```

Comment:
Still needed? If so, check first if language pack already contains PHP files or not.